Automattic/wp-calypso
# Site Migration: Upgrade plan step add dynamic speed based on site LCP

Related to https://github.com/Automattic/wp-calypso/issues/88971

## Proposed Changes

* Changes the text for "Increased Speed" section on the Site migration upgrade plan step. Google considers a Largest Contentful Paint (LCP) of 2.5 seconds or less a "Good" LCP value. If the LCP of the imported site is greater than 2.5 seconds, then we display the updated text with the speed percentage difference

Good LCP: 
<img width="392" alt="CleanShot 2024-05-08 at 11 20 35@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/e2d66963-dcde-4854-891b-c378868eca12">

"Needs improvement/Poor" LCP:

<img width="396" alt="CleanShot 2024-05-08 at 11 28 10@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/4340e75e-fa4e-498c-ae9d-aa4c35412cd7">


## Testing Instructions



Test import sites:
https://www.mazdausa.com - "Slow LCP"
https://www.moneyunder30.com - "Good LCP"

* Checkout branch or use the live link.
* Navigate to both links: `/setup/site-setup/importerWordpress?siteSlug={FREE_SITE_SLUG}&from={IMPORT_SITE_URL}&option=everything` and `/setup/site-migration/site-migration-upgrade-plan?from={IMPORT_SITE_URL}&siteSlug={FREE_SITE_SLUG}`
* Verify updated text appears if LCP is above 2.5 seconds (2500) in API request (`/wpcom/v2/site-profiler/metrics/basic?`)
* Verify speed percentage difference formula is returning correct value.

## Pre-merge Checklist
